PopPUNK/__main__.py

```diff
@@ -405,22 +409,25 @@ def main():
                                             args.score_idx,
                                             args.no_local,
                                             args.gpu_graph)
-                new_model.plot(distMat)
+                if not args.no_plot:
+                    new_model.plot(distMat)
                 model = new_model
             elif args.fit_model == "threshold":
                 new_model = RefineFit(output)
                 new_model.set_threads(args.threads)
                 assignments = new_model.apply_threshold(distMat,
                                                         args.threshold)
-                new_model.plot(distMat)
+                if not args.no_plot:
+                    new_model.plot(distMat)
```

Comment:
Let's just make this a single call at the end of this block for all the models. The only complication is that BGMM currently takes the assignments. It would make sense to add a `y` argument to all the plots in models which is ignored (I think this is already the case for all of them except LineageFit). Then we just have a single:
```
if not args.no_plot:
    model.plot(distMat, assignments)
```
after the if/else

Reply:
Done in 03f11a8.